packages/client/helpers/memory-tests.ts

```diff
@@ -0,0 +1,17 @@
+import { ClientEngineType, getClientEngineType } from '@prisma/internals'
+
+import { runAllMemoryTests } from '../tests/memory/_utils/runAllMemoryTests'
+
+if (getClientEngineType() === ClientEngineType.Binary) {
+  console.log('Memory tests are skipped for binary engine at the moment')
```

Comment:
Binary engine leaks, so I skipped it for now. I think it will make sense to fix it in a separate issue, but it's up for discussion.